packages/forma-36-tokens/tools/build.js

```diff
@@ -1,7 +1,35 @@
 const globby = require('globby');
 const path = require('path');
 const fse = require('fs-extra');
-const _ = require('lodash');
+
+// No real replacement for the _.camelCase, only handles this specific case.
+function toCamelCase(word) {
+  return word
+    .split('-')
+    .map((key, index) =>
+      index === 0 ? key : key.replace(/[A-Z]/i, (match) => match.toUpperCase()),
+    )
+    .join('');
+}
+
+// Maps over object and applies fn in every entry
+function mapObject(obj, fn) {
+  return Object.fromEntries(Object.entries(obj).map(fn));
+}
+
+// Apply toCamelCase on keys of an Object
+function camelCaseKeys(obj) {
+  return mapObject(obj, ([key, value]) => [toCamelCase(key), value]);
+}
+
+function pickBy(obj, predicate) {
+  return Object.fromEntries(Object.entries(obj).filter(predicate));
+}
+
+function findKey(obj, predicate) {
+  const [key] = Object.entries(obj).find((entry) => predicate(entry[1]));
+  return key || undefined;
+}
```

Comment:
👏  👏  👏  👏  👏  👏 

with a couple of simple functions we just avoid importing a big package